### DISCUSSION TITLE
looking for dev who can help us migrate to rainbowkit

### DISCUSSION THREAD

**Discussion by PeerRich** [rainbow-me/rainbowkit#682] [Q&A]
hey everyone, we look for a dev who can join us for a small paid project to replace our existing metamask-only integration with rainbowkit

learn more: https://cal.com/apps/metamask should be replaced with rainbow

current code: https://github.com/calcom/cal.com/blob/464343f5abb1dbc95560498ca7c7a9bfe5a32480/packages/features/ee/web3/components/CryptoSection.tsx

send me an email if you are interested peer@cal.com

**Comment by hexcowboy** [comment]
sent you an email
